.github/workflows/tag-release.yml

```diff
@@ -0,0 +1,44 @@
+name: Tag & GitHub release
+
+on:
+  pull_request_target:
+    branches:
+      - main
+    types: [closed]
+
+jobs:
+  tag-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        if: github.event.pull_request.merged == true
+        with:
+          fetch-depth: 0
+
+      - name: Extract version
+        if: github.event.pull_request.merged == true
+        id: version
+        run: |
+          NEW_VERSION=$(node -p 'require("./package.json").version')
+          echo "::set-output name=version::v$NEW_VERSION"
```

Comment:
Would this be the same as `echo "version=v$NEW_VERSION" >> $GITHUB_OUTPUT` like described [here](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter)?

Reply:
I'll try this syntax